packages/next/next-server/server/load-components.ts

```diff
@@ -32,7 +32,9 @@ type Unstable_getStaticProps = (params: {
   revalidate?: number | boolean
 }>
 
-type Unstable_getStaticPaths = () => Promise<Array<string | ParsedUrlQuery>>
+export type Unstable_getStaticPaths = () => Promise<{
```

Comment:
Does this need to be exported here?

Reply:
It's exported here to allow re-using the type in `build/utils.ts` and only exported for internal use still